keystore.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"github.com/lwithers/minijks/jks"
+	"io/ioutil"
+	"log"
+	"path"
+	"time"
+)
+
+func writeKeystore(certDir string, privkey *rsa.PrivateKey, pemCertificate []byte) {
```

Comment:
Is there anything we could do to Validate/Unit Test the writing of the KeyStore? I understand from the minijks that the format is actually not documented

Reply:
I see two options:
1) Use minijks to read the keystore. That won't catch any consistent error in minijks itself.
2) Add a testing dependency on keytool. That seems a bit too heavyweight.